include/rbdl/rbdl_eigenmath.h

```diff
@@ -12,11 +12,11 @@
  * causes the classes in this file not to be explicitly exported. Instead
  * they are already implicitly exported.
  */
-#if defined(WIN32) && defined(rbdl_EXPORTS)
+//#if defined(WIN32) && ( defined(rbdl_EXPORTS) || defined(rbdl_urdfparser_EXPORTS) || defined(rbdl_luamodel_EXPORTS))
```

Comment:
Do these comments serve a purpose or is that something that slipped during clean-up?